### PR TITLE
Lower-resolution avatars

### PR DESCRIPTION
**Pull request by Sjmarf**

### CODE REVIEW

File: Mlem/App/Views/Shared/Avatar/AvatarView.swift [hunk 1]
```diff
@@ -37,23 +37,25 @@ struct AvatarView: View {
...
-                    .resizable()
-                    .aspectRatio(1, contentMode: .fill)
-                    .background {
-                        if loading {
-                            if showLoadingPlaceholder {
-                                ProgressView()
-                            }
-                        } else {
-                            palette.secondaryBackground
+                VStack {
```
Comment (EricBAndrews) [inline Mlem/App/Views/Shared/Avatar/AvatarView.swift:40]: Why `VStack`?

**Review by EricBAndrews — CHANGES_REQUESTED** [review]
Looks good, small question

File: Mlem/App/Views/Shared/Avatar/AvatarView.swift (hunk 1 above)
Comment (Sjmarf) [reply Mlem/App/Views/Shared/Avatar/AvatarView.swift]: Not needed; I removed it 👍

**Review by EricBAndrews — APPROVED** [review]
LGTM 👍

**Pull request merged**